client/src/components/Cart-dropdown/cart-dropdown.js

```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from "react";
```

Comment:
cart-dropdown component. Where it will display all the products user added. Click the checkout button will take you to the checkout page